artifacts/Artifacts.toml

```diff
@@ -0,0 +1,8 @@
+[land_mask]
```

Comment:
I would have thought it would be good to keep it for tracking changes in the files (each time the file is replaced with a new one of the same name, the `git-tree-sha1 ` changes). I just found [this](https://github.com/JuliaLang/Pkg.jl/issues/1914) issue, which may require using `Tar.tree_hash` to be general to all platforms, but apparently this should have been fixed in Julia 1.6. @charleskawczynski do you have any thoughts? I noticed Atmos doesn't check these in?

Reply:
Yes, I think the artifacts toml is the intended file to be tracked, so that there's a direct mapping between these hashes and the artifacts used.

We don't in ClimaAtmos, but we probably should.

Also, I haven't seen that issue before, it could be that we somehow need to update ArtifactWrappers to use this

Reply:
Thank you, Charlie!